assets/js/services/metadata.js

```diff
@@ -347,6 +347,25 @@ export function prepFieldArrayItemsForPost(items = []) {
   return items.map(({ metadataItem }) => metadataItem);
 }
 
+/**
+ * Convert coded term select options to include scheme
+ * @param {Array} codeListsData Array of object entries
+ * @param {String} codedTerm String representing codedTermScheme
+ * @returns {Array} Array of modified object entries
+ */
+export function getCodedTermSelectOptions(codeListsData, codedTerm) {
```

Comment:
Just a quick error handler, can we give a default values to the params?   

`(codeListsData = [], codedTerm = "")`